src/main/java/frc/robot/subsystems/nDexter.java

```diff
@@ -0,0 +1,161 @@
+        // Set PID values for position control. We don't need to pass a closed loop
+        // slot, as it will default to slot 0.
+        .p(0.1)
+        .i(0)
+        .d(0)
+        .outputRange(-1, 1)
+        // Set PID values for velocity control in slot 1
+        .p(0.0001, ClosedLoopSlot.kSlot1)
+        .i(0, ClosedLoopSlot.kSlot1)
+        .d(0, ClosedLoopSlot.kSlot1)
+        .velocityFF(1.0 / 5767, ClosedLoopSlot.kSlot1)
+        .outputRange(-1, 1, ClosedLoopSlot.kSlot1);
+
+    leftSide.configure(
+        leftMotorConfig, ResetMode.kResetSafeParameters, PersistMode.kNoPersistParameters);
+
+    rightMotorConfig
+        .closedLoop
+        .feedbackSensor(FeedbackSensor.kPrimaryEncoder)
+        // Set PID values for position control. We don't need to pass a closed loop
+        // slot, as it will default to slot 0.
+        .p(0.1)
+        .i(0)
+        .d(0)
+        .outputRange(-1, 1)
+        // Set PID values for velocity control in slot 1
+        .p(0.0001, ClosedLoopSlot.kSlot1)
+        .i(0, ClosedLoopSlot.kSlot1)
+        .d(0, ClosedLoopSlot.kSlot1)
+        .velocityFF(1.0 / 5767, ClosedLoopSlot.kSlot1)
+        .outputRange(-1, 1, ClosedLoopSlot.kSlot1);
+    rightSide.configure(
+        rightMotorConfig, ResetMode.kResetSafeParameters, PersistMode.kNoPersistParameters);
+  }
+
+  public void setTargetVelocity(double leftTarget, double rightTarget) {
+    this.targetVelocityLeft = leftTarget;
+    this.targetVelocityRight = rightTarget;
+  }
+
+  public Command runOnPID() {
+
+    return runOnce(
+        () -> {
+          leftController.setSetpoint(this.targetVelocityLeft);
+          rightController.setSetpoint(this.targetVelocityRight);
+
+          // leftSide.setVoltage(leftController.calculate(leftSide.get()));
+          // rightSide.setVoltage(rightController.calculate(rightSide.get()));
+          rightClosedLoopController.setReference(
+              this.targetVelocityRight,
+              ControlType.kMAXMotionVelocityControl,
+              ClosedLoopSlot.kSlot0);
+          leftClosedLoopController.setReference(
+              this.targetVelocityLeft,
+              ControlType.kMAXMotionVelocityControl,
+              ClosedLoopSlot.kSlot0);
+        });
+  }
+
+  // public boolean overDraw(){
+  //   if(leftSide.getBusVoltage() > MARGIN || rightSide.getBusVoltage() > MARGIN){
+  //     return true;
+  //   }
+  //   return false;
+  // }
+
+  public void assighnPower(double left, double right) {
+    leftSide.setVoltage(left);
+    rightSide.setVoltage(right);
+  }
+
+  public Command power(double left, double right) {
+    return runOnce(() -> this.assighnPower(left, right));
+  }
+
+  public void setTarget(double leftTarget, double rightTarget) {
+    this.targetVelocityLeft = leftTarget;
+    this.targetVelocityRight = rightTarget;
+  }
+
+  public double[] getTargetVelocities() {
+    double ret[] = {this.targetVelocityLeft, this.targetVelocityRight};
```

Comment:
probably replace with a `boolean` to indicate if it can spin in - reference in the spin commands, and add a command to set the inward spinnability